libopenage/util/compiler.cpp

```diff
@@ -73,8 +140,15 @@ std::string symbol_name(const void *addr, bool require_exact_addr, bool no_pure_
 
 bool is_symbol(const void *addr) {
 #ifdef _MSC_VER
-	// TODO: Get dladdr equivalent for MSVC.
-	return true;
+
+	auto symbol_name_result = symbol_name_win(addr);
+	if (symbol_name_result.initialization_fail)
```

Comment:
Oh, I see why the `struct` was defined.  However, if that is changed, this check can still be done by elevating `initialized_symbol_handler_successfully` to the anonymous namespace.